datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -53,6 +53,14 @@ enum EntityType {
     The MLModel Entity
     """
     MLMODEL
+    """
+    The DATAFLOW Entity
+    """
+    DATAFLOW
```

Comment:
Minor nit: Would prefer underscores separating terms for better readability: 

`DATA_FLOW`
`DATA_JOB`